mp/src/game/server/momentum/mom_triggers.h

```diff
@@ -215,14 +214,19 @@ class CBaseMomZoneTrigger : public CBaseMomentumTrigger
 // to have different zone triggers with either similar or different zone numbers.
 class CTriggerZone : public CBaseMomZoneTrigger
 {
-public:
+  public:
     DECLARE_CLASS(CTriggerZone, CBaseMomZoneTrigger);
     DECLARE_DATADESC();
 
     CTriggerZone();
 
     void SetZoneNumber(int newZone) { m_iZoneNumber = newZone; }
-    int GetZoneNumber() const { return m_iZoneNumber; };
+    int GetZoneNumber() const { return m_iZoneNumber; }
+
+    virtual bool IsTeleportableTo() { return false; }
```

Comment:
```suggestion
    bool IsTeleportableTo() const { return GetType() == ZONE_TYPE_START || GetType() == ZONE_TYPE_STAGE; }
```

Reply:
Had to make `GetZoneType()` `const` to do this but it should've already been `const` anyhow.